src/selectors/appStatus.ts

```diff
@@ -11,6 +11,10 @@ export const getAppStatusError = (state: RootStateType): Error | undefined => st
 
 export const getShowEndpointInfoModal = (state: RootStateType): boolean => state.appStatus.showEndpointInfo;
 export const getRunnerEndpoint = (state: RootStateType): { hostname: string; port: string; }  => state.appStatus.endpoint;
+export const getRunnerOrigin = (state: RootStateType): string => {
+	const endpoint = state.appStatus.endpoint;
+	return `http://${endpoint.hostname}:${endpoint.port}`;
```

Comment:
i'll need to pull this to see what the behavior is.. i think this makes sense but we need to note that the runner has a port `5678` (websocket) that doesn't match the HTTP port `3000`